src/lib.rs

```diff
@@ -83,9 +82,9 @@ pub type FunctionKeys = (ProvingKey, VerifyingKey);
 pub fn execute_function(
     function: &Function<Testnet3>,
     user_inputs: &[UserInputValueType],
-    rng: &mut StdRng,
 ) -> Result<(SimpleFunctionVariables, MarlinProof)> {
-    let universal_srs = simpleworks::marlin::generate_universal_srs(rng)?;
+    let mut rng = simpleworks::marlin::generate_rand();
```

Comment:
```suggestion
    let rng = &mut simpleworks::marlin::generate_rand();
```
will let you remove `&mut` when passing as a parameter.